invenio_webhooks/views.py

```diff
@@ -52,6 +52,31 @@
 )
 
 
+def add_link_header(response, links):
+    """Add a Link HTTP header to a REST response.
+
+    :param response: REST response instance.
+    :param links: Dictionary of links.
+    """
+    if links is not None:
+        response.headers.extend({
+            'Link': ', '.join([
+                '<{0}>; rel="{1}"'.format(l, r) for r, l in links.items()])
+        })
+
+
+def make_response(event):
+    """Make a response from webhook event."""
+    response = jsonify(**event.response)
+    response.headers['X-Hub-Event'] = event.receiver_id
```

Comment:
GitHub uses [``X-GitHub-Event``/``X-GitHub-Delivery``](https://developer.github.com/v3/repos/hooks/#webhook-headers) if we want to match the header names with GitHub. They are however inconsistent since they also use ``X-Hub-Signature``.